diff --git a/browser/installer/removed-files.in b/browser/installer/removed-files.in
--- a/browser/installer/removed-files.in
+++ b/browser/installer/removed-files.in
@@ -65,11 +65,14 @@
 # support. This made it possible to delete all of the removal instructions in
 # this file.
 
-# Remove the toplevel chrome.manifest added by bug 1295542.
-#ifdef XP_MACOSX
-@DIR_MACOS@chrome.manifest
-@DIR_RESOURCES@chrome.manifest
-#endif
+# --- Noraneko Patch Start ---
+# This blocks to package with `ac_add_options --enable-chrome-format=flat`
+# # Remove the toplevel chrome.manifest added by bug 1295542.
+# #ifdef XP_MACOSX
+#   @DIR_MACOS@chrome.manifest
+#   @DIR_RESOURCES@chrome.manifest
+# #endif
+# --- Noraneko Patch End   ---
 
 # channel-prefs.js has been removed on macOS.
 #ifdef XP_MACOSX
