diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
index 2b4d35e060..795398058f 100644
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -214,6 +214,11 @@
 @RESPATH@/browser/chrome/icons/default/default128.png
 #endif
 
+; [Noraneko Files]
+@RESPATH@/chrome/noraneko.manifest
+@RESPATH@/chrome/noraneko@JAREXT@
+@RESPATH@/browser/buildid2
+
 ; [DevTools Startup Files]
 @RESPATH@/browser/chrome/devtools-startup@JAREXT@
 @RESPATH@/browser/chrome/devtools-startup.manifest
